getting_started/step_by_step/singletons_autoload.rst

```diff
@@ -114,10 +127,27 @@ loaded!
             var root = get_tree().get_root()
             current_scene = root.get_child( root.get_child_count() -1 )
 
+ .. code-tab:: csharp
+
+    using Godot;
+    using System;
+
+    public class Global : Godot.Node
+    {
+        public Node CurrentScene { get; set; }
+
+        public override void _Ready()
+        {
+            var root = GetTree().GetRoot();
```

Comment:
I think it's better to use `SceneTree root` since the type info is not immediately available from the right side of the assignment.